src/cli/configure/upgrade/index.ts

```diff
@@ -0,0 +1,55 @@
+import { readdir, writeFile } from 'fs/promises';
+import path from 'path';
+
+import { gte, sort } from 'semver';
+
+import { getConsumerManifest } from '../../../utils/manifest';
+import { getSkubaVersion } from '../../../utils/version';
+import type { SkubaPackageJson } from '../../init/writePackageJson';
+import { formatPackage } from '../processing/package';
+
+const getPatches = async (manifestVersion: string): Promise<string[]> => {
+  const patches = await readdir(path.join(__dirname, 'patches'));
+
+  // The patches are sorted by the version they were added from.
+  // Only return patches that are newer or equal to the current version.
+  return sort(patches.filter((filename) => gte(filename, manifestVersion)));
+};
+
+export const upgradeSkuba = async () => {
+  const [currentVersion, manifest] = await Promise.all([
+    getSkubaVersion(),
+    getConsumerManifest(),
+  ]);
+
+  const manifestVersion = (
+    manifest?.packageJson.skuba as SkubaPackageJson | undefined
+  )?.version;
+
+  if (!manifest || !manifestVersion) {
+    throw new Error(
+      'Could not find a skuba manifest, please run `skuba configure`',
```

Comment:
Wonder how many people would be missing it but are genuinely using skuba? Could it gracefully configure automatically?

Reply:
The problem with that is you have to pick a template I believe 🤔